app/views/listings/form/_price.haml

```diff
@@ -33,7 +33,7 @@
                 "#fee_display",
                 "#listing_currency",
                 #{commission_from_seller},
-                #{minimum_commission},
+                #{minimum_commission.format(decimal_mark: ".", symbol: false)},
```

Comment:
Yep, that's the point. We are formatting number for JS to consume and localization messes things up. :)